cytoscape-panzoom.js

```diff
@@ -513,10 +577,41 @@ SOFTWARE.
               return false;
             });
 
+            $button.bind("touchstart", function(e) {
+              e.preventDefault();
+              e.stopPropagation();
+
+              var doZoom = function() {
+                let zoom = cyRef.zoom();
```

Comment:
I haven't checked but I suspect the rest of their code is using 'var' for backwards compatability? If so, advise just using 'var' everywhere to be safe.

Reply:
Don't die little library 😢

Reply:
It be like that sometimes.